changelog.md

```diff
@@ -1,5 +1,8 @@
 # illogical changelog
 
+## 1.2.4
+* Add support for complex references

```

Comment:
As this is public package, let's be more verbose :)
* Add support for array element targeting withing reference operand key
* Add support for array element targeting via reference within reference operand key
* Add support for nested key resolution within reference operand operand key
* Add support for composite key resolution within reference operand operand key

or something like that ....

Reply:
8a58c23a9ba9903ec1f55ead5d206252bc548aa1